apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/utils/tracking.js

```diff
@@ -0,0 +1,26 @@
+// Ensure Tracks Library
+window._tkq = window._tkq || [];
```

Comment:
Can I confirm that `_tkq` is not present on non-production environments? If it _is_ then how can we stop our local work (or testing by a12s) skewing the metrics? 

Also in the past, I've added a mock implementation of the tracking code for non-production envs which simply prints the args to `console`. This is useful for debugging. 

```js
window._tkq = window._tkq || {
    push: (args) => {
     console.log("Tracking event called", args);
   }
};

Reply:
> If it is then how can we stop our local work (or testing by a12s) skewing the metrics?

I'd assume that's what the `_test_` prefix is for?

Reply:
`_test_` is there for us so we can easily filter events made during the development phase when event props might change a lot, we might be sending some wrong data etc… Once things get stable and ready, we remove `_test_` and everything will be tracked under the main event name.

We don't have any concept of environments in this plugin so we would need to introduce and develop something like it in order to filter events. Is this a concern? The whole company uses Calypso in production daily and I don't think we do any sort of tracking filtration.

Reply:
Let's revisit this when it does become a concern.

Reply:
Sounds good. I don't have my `localhost` Jetpack-connected so no tracking happens there.